.github/workflows/ci.yml

```diff
@@ -282,6 +282,56 @@ jobs:
         # npm b/c Yarn 4 doesn't work in Node 12
         run: cd packages/ses && npm run test:platform-compatibility
 
+  hermes-test:
+    name: hermes-test
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20.x]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # without this, setup-node errors on mismatched yarn versions
+      - run: corepack enable
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+
+      - name: Echo node version
+        run: node --version
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      # end macro
+
+      - name: 'build'
+        run: yarn run build
+
+      - name: Fetch Hermes CLI for Linux tar file
+        run: cd packages/ses && mkdir hermes-cli-linux && cd hermes-cli-linux && wget https://github.com/facebook/hermes/releases/download/v0.12.0/hermes-cli-linux-v0.12.0.tar.gz
+
+      - name: Extract Hermes CLI for Linux tar file
+        run: cd packages/ses/hermes-cli-linux && tar -xzvf hermes-cli-linux-v0.12.0.tar.gz
```

Comment:
moving to `packages/ses/package.json` `scripts` could make local setup easier

Reply:
add to or create docs for local debugging

Reply:
https://github.com/facebook/hermes/releases/download/v0.12.0/hermes-engine-cli-v0.12.0.tgz to get started locally

Reply:
nb: the `yarn add -D hermes-engine-cli` way fails to create symlinks in `node_modules/.bin` on macOS (likely Linux/Win too)

<img width="769" alt="Screenshot 2024-07-31 at 3 03 02 PM" src="https://github.com/user-attachments/assets/8424cb78-8983-43fe-b7f1-bee3f77670c0">

Reply:
create shellscript to create appropriate `node_modules/.bin` symlinks

<img width="937" alt="Screenshot 2024-08-01 at 1 31 56 PM" src="https://github.com/user-attachments/assets/38d9ed89-abe8-44a7-a5df-99f98721e8a1">

Linux/Mac are the same, Windows varies slightly but doable (ignoring the `.dll` files)

Reply:
only `npm run <script>` works with `node_modules/.bin`, not `yarn <script>`